components/contact-form/contact-form.js

```diff
@@ -0,0 +1,218 @@
+      },
+      body: JSON.stringify(formData)
+    })
+    .then(res => {
+      this.clearForm()
+      
+      this.setState({ isSent: true })
+      scrollToElement('message-sent')
+    })
+  }
+
+  shouldValidateMessage = () => {
+    this.setState({ validateMessage: true })
+  }
+
+  clearForm = () => {
+    this.setState({
+      name: '',
+      email: '',
+      phoneNumber: '',
+      company: '',
+      message: '',
+      validateMessage: false,
+      selectedItem: '',
+      itemType: '',
+    })
+  }
+
+	render() {
+		const { dropDownOptions, formTitle } = this.props;
+    const { selectedItem, name, company, email, phoneNumber, message, _gotcha, validateMessage, itemType, isSent } = this.state;
+    const { handleClick, handleChange, shouldValidateMessage, handleSubmit } = this
+    const messageInputClass = validateMessage ? 'should-validate' : ''
+    
+    if (!isSent) {
+      return (
+        <div className='contact-form container'>
+          <h2 className='form-title'>{formTitle}</h2>
+          
+          <SelectDropdown 
+            dropDownOptions={dropDownOptions} 
+            handleClick={handleClick} 
+            selectedItem={selectedItem}
+          />
+          
+          {(itemType === 'personal' || itemType === 'business') && 
+          <div>
+            <form className='form' onSubmit={handleSubmit}>
+              <InputField
+                name='name'
+                label='My name is'
+                type='text'
+                onChange={handleChange}
+                value={name}
+              />
+
+              { (itemType === 'business') &&
+              <InputField
+                name='company'
+                label='My company is'
+                type='text'
+                onChange={handleChange}
+                value={company}
+              />
+              }
+
+              <InputField
+                name='email'
+                label='My email is'
+                type='email'
+                onChange={handleChange}
+                value={email}
+              />
+
+              { (itemType === 'business') &&
+              <InputField
+                name='phoneNumber'
+                label='My phone number is'
+                type='tel'
+                onChange={handleChange}
+                value={phoneNumber}
+              />
+              }
+
+          <div className='input-field message-input'>
+            <label className='label' htmlFor='message'>My message is</label>
+            
+            <textarea 
+              id='message'
+              className={`input textarea ${messageInputClass}`}
+              name='message'
+              value={message}
+              onChange={handleChange}
+              required
+              onBlur={shouldValidateMessage}
+            />
+          </div>
+          
+          <input type="text" name="_gotcha" value={_gotcha} style={{ display: 'none' }} onChange={handleChange} />
```

Comment:
consider making this a `type="hidden"`, since a screen reader could pick this up. If you are concerned that scrapers/bots skip it because of the `type="hidden"`, they also will be smart enough to see the `display: none;`...